plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -82,7 +82,13 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
 		// in case of latency in our caches, make a call direct to storage to verify that it truly exists or not
 		namespaceObj, err = l.client.Namespaces().Get(a.GetNamespace())
 		if err != nil {
-			return admission.NewNotFound(a)
+			if errors.IsNotFound(err) {
```

Comment:
@brendandburns recently made a change to return a 404 when attempting to put something in a Namespace that does not exist.  I don't want to change that behavior to now return a Forbidden again.  I think the current code here is an improvement in that it tells you that the Namespace itself is not present.  The ```NamespaceExists``` plugin is deprecated but the same change should be made there.